backend/de.metas.purchasecandidate.base/src/main/java/de/metas/purchasecandidate/purchaseordercreation/remoteorder/RealVendorGatewayInvoker.java

```diff
@@ -154,6 +151,19 @@ public List<PurchaseItem> placeRemotePurchaseOrder(
 		return result.build();
 	}
 
+	@NonNull
+	private OrgId getOrgIdFromPurchaseCandidates(final Collection<PurchaseCandidate> purchaseCandidates)
+	{
+		final OrgId orgId = purchaseCandidates.stream()
+				.map(PurchaseCandidate::getOrgId)
+				.reduce((a, b) -> {
+					throw new AdempiereException("Can only process purchase candidates from a single organization.");
```

Comment:
ok, FTR, here i wanted to write "please give more infos so that the user knows just *which* candidates this is about
but i guess that's clear, from the workpackage